packages/docs-reanimated/docs/layout-animations/layout-transitions.mdx

```diff
@@ -4,195 +4,376 @@ sidebar_position: 3
 
 # Layout transitions
 
-import DocsCompatibilityInfo from '../_shared/_docs_compatibility_info.mdx';
+Layout transition allows you to animate changes in the layout of elements.
 
-<DocsCompatibilityInfo />
+import LayoutTransitions from '@site/src/examples/LayoutTransitions';
+import LayoutTransitionsSrc from '!!raw-loader!@site/src/examples/LayoutTransitions';
+import ExampleVideo from '@site/src/components/ExampleVideo';
 
-The document explains how can you animate all layout changes for a specific view just by adding a single property to the view.
-To be precise how to animate positions and dimensions of components. What's important it will all happen entirely on UI thread without any communication through the bridge. There are plenty of ways in which you can animated layout changes however in contrast to entering and exiting animations they are not so regular. We've prepared a few predefined layout transitions, however if you want to create more custom and complex transition you can create your own.
+<InteractiveExample src={LayoutTransitionsSrc} component={LayoutTransitions} />
 
-## How to use predefined layout transition?
+## Predefined Transitions
+
+Below we listed all of the currently available predefined layout transitions with their [layout animation modifiers](/docs/fundamentals/glossary#layout-animation-modifier).
+
+## Linear transition
 
-### 1. Import chosen transition
+Linear transition animates both position and dimension in the same way.
+
+### Reference
 
 ```js
-// Transition is just an example and should be replaced by real animation. For Instance Layout
-import { Transition } from 'react-native-reanimated';
+import { LinearTransition } from 'react-native-reanimated';
+
+function App() {
+  return <Animated.View layout={LinearTransition} />;
+}
 ```
 
-### 2. Choose Animated Component which layout you want to animate
+### Modifiers
 
-```js
-    // AnimatedComponent - component created by createAnimatedComponent or imported from Reanimated
-    <AnimatedComponent layout={Transition} >
+#### Time-based <Optional/>
+
+Time-based modifiers relay on [`withTiming`](/docs/animations/withTiming) function.
+
+```javascript
+LinearTransition.duration(500).easing(Easing.ease);
 ```
 
-### 3. Customize the animation
+- `.easing(easingFunction: EasingFunction)` is an easing function which defines the animation curve. Defaults to `Easing.inOut(Easing.quad)`
+
+:::note
+Time-based modifiers have no effect when `.springify()` is used.
+:::
+
+#### Spring-based <Optional/>
```

Comment:
See withSpring documentation for the list of modifiers: https://docs.swmansion.com/react-native-reanimated/docs/animations/withSpring
Generally usage of any of time-based spring modifiers erases the effect of any physics-based spring modifier, so I have a feeling that it would look bit more natural to these groups as two columns next to each other. And the warning should be here instead of inside remarks

Reply:
I think to be consistent, the list of LinearTransition modifiers should stay as a list, as it could 'mislead' the user - user has access by link to see withSpring() config where it is grouped into more informative tables - what do you think?

Reply:
Sure